src/v1/examples/LayeredPenguins.sol

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.25;
+
+import {LPNClientV1} from "../client/LPNClientV1.sol";
+import {ILPNRegistryV1} from "../interfaces/ILPNRegistryV1.sol";
+import {
+    ERC721Enumerable,
+    ERC721
+} from "@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol";
+import {L1BlockNumber} from "../../utils/L1Block.sol";
+import {QueryOutput} from "../Groth16VerifierExtensions.sol";
+
+/// @notice Refer to docs page https://lagrange-labs.gitbook.io/lagrange-v2-1/zk-coprocessor/testnet-euclid-developer-docs/example-nft-mint-whitelist-on-l2-with-pudgy-penguins
+contract LayeredPenguins is LPNClientV1, ERC721Enumerable {
+    /// SELECT key FROM pudgy_penguins_owners WHERE value = $1;
```

Comment:
We only support aggregate function - can you make your example work with one ? If you do average over 1 block I guess it works ahah.

Reply:
Yeah I'll do that ... do we also need to update Parsil to reject queries that are not aggregates ?

Reply:
Yup, good point, we're doing it asap.